README.md

```diff
@@ -1,4 +1,106 @@
+#### 3. Bot
+Now we have setup our repository we need somewhere to test out bot.
+You'll need to make a new discord server:
+![](https://i.imgur.com/49gBlQI.png)
+We need to make the applicaiton for our bot... navigate over to [discordapp.com/developers](https://discordapp.com/developers) and hit new application
+![](https://i.imgur.com/UIeGPju.png)
+Now we have our discord application you'll want to name your bot as below:
+![](https://i.imgur.com/odTWSMV.png)
+To actually make the bot hit Bot->Add Bot->Yes, do It! as below:
+![](https://i.imgur.com/frAUbTZ.png)
+Copy that Token and put to somewhere for safe keeping.
+![](https://i.imgur.com/oEpIqND.png)
+Now to add that robot to out new discord server we need to generate an OAuth2 Url to do so navigate to the OAuth2 tab, Scrollto the OAUTH2 URL GENERATOR section, click the Bot checkbox in the scope section and finally hit the administrator checkboxin the newly formed Bot Permissions section.
+![](https://i.imgur.com/I2XzYPj.png)
+Copy and paste the link into your browser and follow the instructions to add the bot to your server - ensure it is the server you have just created.
+#### 4. Run Configurations
+Go back to PyCharm and you should have something a bit like below, Your going to want to hit the Add Configuration button in the top right.
+![](https://i.imgur.com/nLpDfQO.png)
+We are going to want to choose a python config as below:
+![](https://i.imgur.com/9FgCuP1.png)
+The first setting we need to change is script path as below (the start script may have changed from bot.py so be sure to click the right one
+![](https://i.imgur.com/napKLar.png)
+Now we need to add an enviroment variable - what this will do is allow us to set a value without it affact the main repository.
+To do this click the folder icon to the right of the text, then on the new window the plus icon. Now name the var `HACKTOBERBOT_TOKEN` and give the value the token we kept for safe keeping earilier.
+![](https://i.imgur.com/nZFWNaQ.png)
+Now hit apply on that window and your ready to get going!
+#### 5. Git in PyCharm
+As we work on our project we are going to want to make commits. Commits are effectively a list of changes you made to the pervious version. To make one first hit the green tick in the top right
+![](https://i.imgur.com/BCiisvN.png)
+1. Select the files you wish to commit
+2. Write a brief description of what your commit is
+3. See the actual changes you commit does here (you can also turn some of them off or on if you wish)
+4. Hit commit
+
+![](https://i.imgur.com/xA5ga4C.png)
+Now once you have made a few commits and are happy with your changes you are going to want to push them back to your fork.
+There are three ways of doing this.
+1. using the VSC Menu `VSC->Git->Push
+2. using the VSC popup <code>alt-\`->Push</code>
+3. or a shortcut: `ctrl+shift+K`
+
+You should get a menu like below:
+1. list of commits
```

Comment:
be consistent with leading capital letters.

Reply:
(there are a couple of these spread across this whole document, please go through it and fix them all)